Microsoft.DotNet.Interactive.Tests/LanguageKernelTests.cs

```diff
@@ -1576,5 +1576,27 @@ public async Task issue_637()
                   .ContainSingle(e => ((PackageAdded)e).PackageReference.PackageName == "Microsoft.NETCore.App.Ref");
 
         }
+
+        [Fact]
+        public async Task issue_624()
```

Comment:
More descriptive test name.  You can link to the fixed issue number in the attribute.